.gitlab/integration_test/otel.yml

```diff
@@ -21,3 +21,29 @@ integration_tests_otel:
     - if: $CI_COMMIT_REF_NAME =~ /.*-skip-cancel$/
       when: never
     - when: always
+
+
+docker_image_build_otel:  
+  stage: integration_test
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  needs: ["go_deps","integration_tests_otel"]
+  tags: ["runner:docker"]
+  before_script:
+    - !reference [.retrieve_linux_go_deps]
+    - mkdir -p /tmp/otel-ci
+    - cp comp/otelcol/collector-contrib/impl/manifest.yaml /tmp/otel-ci/
```

Comment:
can u modify manifest.yaml , by adding a component not present in the default build ? either metricstransform processor or something else